src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-standard.md

```diff
@@ -0,0 +1,209 @@
+  - `nuxeo.retention.s3storage.awsid`</br>
+    (fallback on `nuxeo.s3storage.awsid`)
+  - `nuxeo.retention.s3storage.awssecret`</br>
+    (fallback on `nuxeo.s3storage.awssecret`)
+  - `nuxeo.retention.s3storage.awstoken`</br>
+    (fallback on `nuxeo.s3storage.awstoken`)
+  - `nuxeo.retention.s3storage.region`</br>
+    (fallback on `nuxeo.s3storage.region`)
+  - `nuxeo.retention.core.blobstore.digestAsync`</br>
+    (fallback on `nuxeo.core.blobstore.digestAsync`)
+  - `nuxeo.retention.s3storage.cacheminage`</br>
+    (fallback on `nuxeo.s3storage.cacheminage`)
+  - `nuxeo.retention.s3storage.cachesize`</br>
+    (fallback on `nuxeo.s3storage.cachesize`)
+  - `nuxeo.retention.s3storage.cachecount`</br>
+    (fallback on `nuxeo.s3storage.cachecount`)
+  - `nuxeo.retention.s3storage.connection.max`</br>
+    (fallback on `nuxeo.s3storage.connection.max`)
+  - `nuxeo.retention.s3storage.connection.retry`</br>
+    (fallback on `nuxeo.s3storage.connection.retry`)
+  - `nuxeo.retention.s3storage.connection.timeout` </br>
+    (fallback on `nuxeo.s3storage.connection.timeout`)
+  - `nuxeo.retention.s3storage.socket.timeout` </br>
+    (fallback on `nuxeo.s3storage.socket.timeout`)
+  - `nuxeo.retention.s3storage.endpoint`</br>
+    (fallback on `nuxeo.s3storage.endpoint`)
+  - `nuxeo.retention.s3storage.pathstyleaccess`</br>
+    (fallback on `nuxeo.s3storage.pathstyleaccess`)
+  - `nuxeo.retention.s3storage.accelerateMode`</br>
+    (fallback on `nuxeo.s3storage.accelerateMode`)
+
+
+#### {{> anchor 's3-one-bucket-object-lock'}} Store the Records in a Dedicated S3 Bucket, With S3 Object Lock In Governance Mode
+
+Using [Amazon S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lock.html) allows to secure the content under retention by preventing to delete a blob into S3 until the retention period has expired.
+
+If you configure your S3 bucket with Amazon S3 Object Lock, Nuxeo will automatically provide the retention period (or the legal hold) to Amazon S3 to apply the retention at storage level.
+
+{{#> callout type='warning'}}
+Due to the capability to [delete a document under retention]({{page page='nuxeo-retention-functional-overview'}}#delete-document-under-retention), the Retention addon in standard mode supports [Amazon S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lock.html) only in Governance mode.
+{{/callout}}
+
+To configure the Retention addon with a dedicated S3 bucket with Object lock, you have to follow the exact same steps than described in the  [previous section](#s3-2-buckets-configuration).
+
+In addition, you have to configure the S3 bucket dedicated to the records as follow:
+
+- Direct writes to the Amazon S3 storage system must be disabled, to ensure all documents transit through Nuxeo Platform for compliant processing.
+
+- The Amazon S3 Object Lock feature must be enabled in Governance Mode on the bucket intended to store objects that are marked as final records.
```

Comment:
better do not define any default retention policy at the bucket level. Let Nuxeo app set the proper retention for each binary